coffee-chats/src/test/java/com/google/step/coffee/data/UserStoreTest.java

```diff
@@ -0,0 +1,56 @@
+package com.google.step.coffee.data;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.users.User;
+import com.google.step.coffee.TestHelper;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UserStoreTest extends TestHelper {
+
+  final String testUserId = "id1234test";
+  final String testEmail = "test-user@example.com";
+
+  UserStore userStore = new UserStore();
+
+  @Test
+  public void addingNewUserHasCorrectEmail() {
+    User user = mock(User.class);
+    when(user.getUserId()).thenReturn("id1234test");
+    when(user.getEmail()).thenReturn("test-user@example.com");
```

Comment:
Why repeat yourself if `testUserId` and `testUserEmail` exist?

Reply:
Fixed.